helm-charts/splunk-otel-collector/values.yaml

```diff
@@ -240,7 +240,14 @@ otelAgent:
       # This value is being used as a source for default memory_limiter processor configurations
       memory: 500Mi
 
-  securityContext: {}
+  # Boolean for running initContainer that will change group ownership of directories for container logs and journald logs.
+  # It is required for running the agent as a non root user. By default, it will run as UID and GID 20000.
+  # Disable it if you would rather run it as root user.
+  runInitForNonRoot: false
```

Comment:
Do we need this option?

Maybe we can rely on `securityContext.runAsUser` to be anything other than `0` instead? 

Then we can have
```
#  securityContext:
#    runAsUser: 20000
#    runAsGroup: 20000
```
as a recommended commented out config.

Reply:
Otherwise any custom setting of `securityContext` that user might have will be ignored with this change by default